src/Shared/UnitTests/MockLogger.cs

```diff
@@ -416,7 +416,7 @@ static private ResourceManager EngineResourceManager
             {
                 if (s_engineResourceManager == null)
                 {
-                    s_engineResourceManager = new ResourceManager("Microsoft.Build.Resources.Strings", typeof(ProjectCollection).GetTypeInfo().Assembly);
+                     s_engineResourceManager = new ResourceManager("Microsoft.Build.Strings", typeof(ProjectCollection).GetTypeInfo().Assembly);
```

Comment:
The new convention is `<AssemblyName>.<NeutralResxName>.resources`: https://github.com/Microsoft/msbuild/blob/master/build/LocalizationTasks/EmitLocalizedResources.cs#L72

For this diff, the resource manager most likely won't be able to find `Microsoft.Build.Strings`.